backend/api/Controllers/InspectionFindingsController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Api.Controllers.Models;
+using Api.Database.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers
+{
+    [ApiController]
+    [Route("inspection-findings")]
+    public class InspectionFindingsController : ControllerBase
+    {
+        private readonly IInspectionService _inspectionService;
+        private readonly ILogger<InspectionFindingsController> _logger;
+        public InspectionFindingsController(
+            ILogger<InspectionFindingsController> logger,
+            IInspectionService inspectionService
+        )
+        {
+            _logger = logger;
+            _inspectionService = inspectionService;
+
+        }
+
+        /// <summary>
+        /// Associate a new inspection finding with the inspection corresponding to isarStepId
+        /// </summary>
+        /// <remarks>
+        /// </remarks>
+        [HttpPost("add-findings")]
+        [Authorize(Roles = Role.Admin)]
+        [ProducesResponseType(typeof(InspectionFindingsResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status409Conflict)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public async Task<ActionResult<InspectionFindings>> AddFindings([FromBody] InspectionFindingsQuery inspectionFinding)
```

Comment:
Lines 32 says it produces output of type InspectionFindingsResponse while the return type on line 38 says it returns InspectionFindings, this is incorrect yeah?

Reply:
Yes it was incorrect. Fixed now I believe.